### PR TITLE
PXP-7296 PXP-7180 Feat/va gwas app

### PR DESCRIPTION
**Pull request by mfshao**
This PR fulfills https://ctds-planx.atlassian.net/browse/PXP-7296 and https://ctds-planx.atlassian.net/browse/PXP-7180

Complete VA GWAS app by adding the following features
1. Connect with workspace storage service
2. Post new jobs to mariner
3. Monitor mariner job status

 Deployed at https://qa-mickey.planx-pla.net/analysis/vaGWAS (make sure you have `mariner_admin` policy in user.yaml in order to submit mariner jobs)

### New Features
- The new VA GWAS app is implemented
- VA GWAS app now talks to workspace storage service for handling files
- VA GWAS app now talks to mariner for submitting new GWAS jobs
- VA GWAS app now has a progress tracking component to visualize mariner job status and display logs

### Deployment changes
- Added new entry `GWASApp` for `analysisTools` for this new GWAS app for VA. Beaware not to confuse with the old "GWAS PoC" app, which bears the entry `vaGWAS`
- `antd modal` now by default will use `g3-color__base-blue` as header background color, this is consistant with our color schemes for other UI components. If an env has customized color scheme, please override `.ant-modal-header` in `gitops.css`

### CODE REVIEW

File: src/Analysis/VAGWAS.jsx [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+                <List
+                  className='vaGWAS__jobStatusList'
+                  itemLayout='horizontal'
+                  pagination={{ pageSize: 5 }}
+                  dataSource={this.state.marinerJobStatus}
+                  renderItem={item => (
+                    <List.Item
+                      actions={(item.status === 'running') ?
+                        [<Popconfirm
+                          title='Are you sure to cancel this job?'
```
Comment (ZakirG) [inline src/Analysis/VAGWAS.jsx:639]: maybe better phrased "Are you sure you want to cancel this job?"

File: src/localconf.js [hunk 2]
```diff
@@ -101,11 +101,15 @@ function buildConfig(opts) {
...
-  const guppyGraphQLUrl = `${guppyUrl}/graphql/`;
+  const guppyGraphQLUrl = `${guppyUrl}/graphql`;
   const guppyDownloadUrl = `${guppyUrl}/download`;
   const manifestServiceApiPath = typeof manifestServiceURL === 'undefined' ? `${hostname}manifests/` : ensureTrailingSlash(manifestServiceURL);
   const requestorPath = typeof requestorURL === 'undefined' ? `${hostname}requestor/` : ensureTrailingSlash(requestorURL);
   const auspiceUrl = `${protocol}//auspice.${hostnameOnly}/covid19`;
+  const workspaceStorageUrl = `${hostname}ws-storage`;
+  const workspaceStorageListUrl = `${workspaceStorageUrl}/list`;
+  const workspaceStorageDownloadUrl = `${workspaceStorageUrl}/download`;
+  const marinerUrl = `${hostname}/ga4gh/wes/v1/runs`;
```
Comment (ZakirG) [inline src/localconf.js:112]: I believe this variable contains two slashes in a row -- it ends up looking like this: https://qa-mickey.planx-pla.net//ga4gh/wes/v1/runs

Comment (mfshao) [reply src/localconf.js:112]: oops good catch!

**Comment by ZakirG** [comment]
Optional suggestion: It might be helpful to future users to be informed in the UI if they lack the mariner_admin policy. Right now, that case results in a console error with a 403 from the mariner URL.

**Comment by mfshao** [comment]
> Optional suggestion: It might be helpful to future users to be informed in the UI if they lack the mariner_admin policy. Right now, that case results in a console error with a 403 from the mariner URL.

Yeah that is a good point, and I had thought about it. I agree this is useful, but probably not in the scope of these 2 tickets. However I can create a ticket for this and prioritize it later.

**Comment by ZakirG** [comment]
Some assorted feedbacks:
- The "Previous" and "Next" buttons at the bottom of the page are misaligned
- I think those buttons should be a bit bigger

Overall, amazing interface. Best UI that exists in data-portal so far I think ~

**Review by ZakirG — APPROVED** [review]

**Comment by mfshao** [comment]
@paulineribeyre feels I should tag you since the `Deployment changes` of this PR will affect the look and feel of NCT

**Pull request merged**